packages/search-ui/src/actions/trackAutocompleteClickThrough.js

```diff
@@ -8,6 +8,10 @@
  * this click event
  */
 export default function trackAutocompleteClickThrough(documentId, tags = []) {
+  if (this.debug)
+    // eslint-disable-next-line no-console
+    console.log("Action", "trackAutocompleteClickThrough", ...arguments);
```

Comment:
This debug/console is different than all the rest. 🤷🏻‍♂️

Reply:
@omegahm It was just because it was automatically wrapped to 2 lines by my editor, due to it's length.

I reformatted it so it looks a bit better.

While I was updating the code, I also changes the state logging to log both the state that changed AND the complete state.

Mind re-approving this?